clang/docs/ReleaseNotes.rst

```diff
@@ -218,6 +218,9 @@ Bug Fixes in This Version
   (`#65156 <https://github.com/llvm/llvm-project/issues/65156>`_`)
 - Clang no longer considers the loss of ``__unaligned`` qualifier from objects as
   an invalid conversion during method function overload resolution.
+- Clang now properly handles out of line template specializations when there is
+  a non-template inner-class between the function and the class template
+  (`#65810 <https://github.com/llvm/llvm-project/issues/65810>`_).
```

Comment:
```suggestion
  a non-template inner-class between the function and the class template.
  (`#65810 <https://github.com/llvm/llvm-project/issues/65810>`_)
```

For consistency (Yes, it's weird). Can you also move that to the "Bug Fixes to C++ Support" section?